README.md

```diff
@@ -1,57 +1,29 @@
-# Project Name
+# Apple.com Webpage Clone
 
-> One paragraph statement about the project.
+> This project was to clone the Apple.com webpage using backgrounds and gradient colors.
 
-![screenshot](./app_screenshot.png)
+![screenshot](/images/apple-webpage.png)
```

Comment:
# [REQUIRED] - FIX - Link to a screenshot.

Change your link as shown below to allow anyone viewing your PR see the screenshot.
```suggestion
![screenshot](./images/apple-webpage.png)
```

This is what I see: 
![screenshot](https://user-images.githubusercontent.com/22345961/78175809-24bae080-7464-11ea-8b7d-e4e149f2959e.png)